src/main/webapp/itempagescript.js

```diff
@@ -24,11 +22,7 @@ async function createSelectedItemCard(entertainmentItem) {
   card.append(
     $('<img class="card-img-top" src="' + entertainmentItem.imageUrl + '">'));
   const cardBody = $('<div class="card-body"></div>');
-  cardBody.append(
-    $('<h5 class="card-title">' + entertainmentItem.title + '</h5>'));
```

Comment:
Why did you remove the title?

Reply:
This code is outdated and for some reason, my pushes aren't affecting the git